src/main/java/core/basesyntax/utils/convert/csv/CsvConverterImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.utils.convert.csv;
+
+import core.basesyntax.utils.convert.ReportConverter;
+import core.basesyntax.utils.transaction.FruitTransaction;
+import core.basesyntax.utils.transaction.OperationFactory;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class CsvConverterImpl implements ReportConverter {
+    private static final String LINE_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int PRODUCT_INDEX = 1;
+    private static final int PRODUCT_QUANTITY_INDEX = 2;
+    private final String report;
```

Comment:
Storing the entire report as a String inside the converter is not the best approach. It would be better to pass the report content directly to the `convert` method, which would make this class stateless and more aligned with SOLID principles.